shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/test/java/org/apache/shardingsphere/data/pipeline/spi/ddlgenerator/DialectDDLGeneratorFactoryTest.java

```diff
@@ -0,0 +1,48 @@
+package org.apache.shardingsphere.data.pipeline.spi.ddlgenerator;
```

Comment:
Please add license information in the file header, refer to other files.

Reply:
I'm solved.